native/src/components/TimeStamp.tsx

```diff
@@ -14,36 +14,38 @@ const TimeStampText = styled.Text`
 type DirectionContainerPropsType = {
   language: string
   children: React.ReactNode
-  theme: ThemeType
 }
+
 const DirectionContainer = styled.View<DirectionContainerPropsType>`
   display: flex;
   flex-direction: ${props => contentDirection(props.language)};
 `
 type PropsType = {
   lastUpdate: Moment
   formatter: DateFormatter
-  t: TFunction
-  language: string
   theme: ThemeType
   showText?: boolean
   format?: string
 }
 
-export class TimeStamp extends React.PureComponent<PropsType> {
-  render(): ReactNode {
-    const { lastUpdate, formatter, t, language, theme, showText = true, format = 'LL' } = this.props
-    // only show day, month and year
-    const dateText = formatter.format(lastUpdate, {
-      format
-    })
-    return (
-      <DirectionContainer language={language} theme={theme}>
-        {showText && <TimeStampText theme={theme}>{t('lastUpdate')} </TimeStampText>}
-        <TimeStampText theme={theme}>{dateText}</TimeStampText>
-      </DirectionContainer>
-    )
-  }
+export const TimeStamp = ({
+  lastUpdate,
+  formatter,
+  theme,
+  showText = true,
+  format = 'LL'
+}: PropsType): ReactElement => {
+  const { i18n, t } = useTranslation('common')
+  // only show day, month and year
+  const dateText = formatter.format(lastUpdate, {
+    format
+  })
+  return (
+    <DirectionContainer theme={theme} language={i18n.language}>
```

Comment:
passing `i18n.language` will fix the alignment of Last Update, when system language is ltr and content language is rtl.

Reply:
@Taggotty could you check the news list as well please? Seems like we have the same problem there on native

Reply:
![Uploading Simulator Screen Shot - iPhone 12 Pro Max - 2021-08-15 at 11.13.28.png…]()

Reply:
Hmmm. Cannot reproduce this. Does it only happen for local news?

Reply:
Don't know, then just ignore it for now and I will look into it. Thx for checking :)